### ISSUE TITLE
Eliminate TestRequestBuilder._load_data()

### ISSUE THREAD

**Issue by hannes-ucsc** [DataBiosphere/azul#278]
The indirection of putting test input and expected output into separate JSON files to be loaded by the test is driving me nuts. It makes the tests considerably harder to read and comprehend because they are now self-contained.

Example:

```patch
Index: test/service/test_request_builder.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- test/service/test_request_builder.py	(revision 0ff78bb6fc6c40c4e65cf8027ff5b59c793f9813)
+++ test/service/test_request_builder.py	(date 1535678779000)
@@ -16,7 +16,29 @@
     def setUpClass(cls):
         super().setUpClass()
         cls.data_directory = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
-        cls.request_config_filepath = os.path.join(cls.data_directory, 'request_builder_test_config.json')
+        cls.request_config = {
+            "translation": {
+                "entity_id": "entity_id",
+                "entity_version": "entity_version"
+            },
+            "autocomplete-translation": {
+                "files": {
+                    "entity_id": "entity_id",
+                    "entity_version": "entity_version"
+                },
+                "donor": {
+                    "donor": "donor_uuid"
+                }
+            },
+            "manifest": [
+                "File ID:Version",
+                "Assay Id",
+                "Analysis Id",
+                "Project Id"
+            ],
+            "facets": [
+            ]
+        }
 
     def _load_json(self, name):
         return EsTd.open_and_return_json(os.path.join(os.path.dirname(__file__), name))
@@ -45,9 +67,28 @@
         Tests creation of a simple request
         :return: True or False depending on the assertion
         """
-        # Load files required for this test
-        request_config = self._load_json(self.request_config_filepath)
-        expected_output = self._load_json(os.path.join(self.data_directory, 'request_builder_test_input1.json'))
+        expected_output = {
+            "post_filter": {
+                "bool": {
+                    "must": [
+                        {
+                            "constant_score": {
+                                "filter": {
+                                    "terms": {
+                                        "entity_id.keyword": [
+                                            "cbb998ce-ddaf-34fa-e163-d14b399c6b34"
+                                        ]
+                                    }
+                                }
+                            }
+                        }
+                    ]
+                }
+            },
+            "query": {
+                "match_all": {}
+            }
+        }
         # Create a simple filter to test on
         sample_filter = {"entity_id": {"is": ["cbb998ce-ddaf-34fa-e163-d14b399c6b34"]}}
         # Need to work on a couple cases:
@@ -61,7 +102,7 @@
         es_search = EsTd.create_request(
             sample_filter,
             es_ts_instance.es_client,
-            request_config,
+            self.request_config,
             post_filter=True)
         # Convert objects to be compared to strings
         expected_output = json.dumps(
```

And then delete the darn JSON files.

We use a similar pattern for some of the indexer tests, but those JSONs are much larger. 

Also consider compressing the literals a bit by putting short lists and dicts onto a single line.

┆Issue is synchronized with this [JIRA Story](https://ucsc-cgl.atlassian.net/browse/AZUL-183)
┆Project Name: azul
┆Issue Number: AZUL-183

**Comment by hannes-ucsc** [comment]
Hannes to verify if this can be closed.

**Comment by hannes-ucsc** [comment]
Fixed in 14f6cd317779db19109ecb266e3b1ea1eb995842 and 0f65bb695ba64da6971b26229b57720a869dff06.
